### ISSUE TITLE
A11y: Input - Add intl props for calcite-input internal buttons

### ISSUE THREAD

**Issue by driskull** [Esri/calcite-components#2920]
### Summary

The input component needs aria-labels for the clear/increase/decrease buttons. To do so, we need to add `intl` props so users can set these.

### Actual Behavior

Number buttons are not labeled.

### Expected Behavior

Number buttons are labeled.

### Reproduction Steps

1.

### Relevant Info 

```
    const inputClearButton = (
      <button
        aria-label="todo: this needs an intl prop for a11y."
        class={CSS.clearButton}
        disabled={this.disabled || this.readOnly}
        onClick={this.clearInputValue}
@@ -627,6 +632,7 @@ export class CalciteInput {

    const numberButtonsHorizontalUp = (
      <button
        aria-label="todo: this needs an intl prop for a11y."
        class={{
          [CSS.numberButtonItem]: true,
          [CSS.buttonItemHorizontal]: isHorizontalNumberButton
@@ -643,6 +649,7 @@ export class CalciteInput {

    const numberButtonsHorizontalDown = (
      <button
        aria-label="todo: this needs an intl prop for a11y."
        class={{
          [CSS.numberButtonItem]: true,
          [CSS.buttonItemHorizontal]: isHorizontalNumberButton
```

**Comment by jcfranco** [comment]
I'd like to propose removing the stepper buttons from the tab order. Keyboard users will most likely use the ⬆️ ⬇️  keys to nudge the values and not go through the increase/decrease buttons. This aligns with the way `<input type=number>` behaves. The following component libs work similarly:

* https://opensource.adobe.com/spectrum-web-components/components/number-field/
* https://explore.fast.design/components/fast-number-field
* https://www.carbondesignsystem.com/components/number-input/code
* https://lion-web.netlify.app/components/inputs/input-stepper/overview/

**Comment by driskull** [comment]
I'm ok with that proposal.

**Comment by jcfranco** [comment]
You are time-traveling fast! 🚀 The proposed changes already landed in https://github.com/Esri/calcite-components/pull/2609/. 

Closing!
